lib/inspec/fetcher/url.rb

```diff
@@ -120,6 +128,38 @@ def to_s
 
     private
 
+    class << self
+      def default_ref(match_data)
+        remote_url = "https://github.com/#{match_data[:user]}/#{match_data[:repo]}.git"
```

Comment:
Here is where default ref assumes a github address.

Reply:
could we just make the whole URI .. what if it is a git on gitlab ... like gitlab.mitre.org?

Reply:
we have the whole uri of the repo when we put it in the cli command right?

Reply:
and we put the whole uri in the `inpsec.yml` - basically every time we reference a git repo we do so via a full uri right?

Reply:
The existing code was applying transformations to fetch tarballs, if possible, by recognizing github and bitbucket URLs. We can't do that in a generic basis, as we don't know what transformation may need to be applied. There may be a transformation that works for gitlab, but the user can always just supply the correct tarball URL in the first place.

As this is a hotfix PR, to fix an urgent problem, we will not be expanding any functionality on this PR. If you'd like to explore expanding the transformation logic, feel free to do so, but we need to ship this fix.

Reply:
Ok. I will open an issue, as I know we have customers using InSpec with other git platforms - GitLab mostly - which we should allow for. In theory, I would have guessed that given the user has to give the full git repo URI that we could use that.